package/batocera/emulationstation/batocera-emulationstation/controllers/es_input.cfg

```diff
@@ -3800,4 +3800,27 @@
 		<input name="x" type="button" id="2" value="1" code="307" />
 		<input name="y" type="button" id="3" value="1" code="308" />
 	</inputConfig>
+	<inputConfig type="joystick" deviceName="StadiaNT2K-2088" deviceGUID="05000000d11800000094000000010000">
+		<input name="a" type="button" id="4" value="1" code="305" />
+		<input name="b" type="button" id="3" value="1" code="304" />
+		<input name="down" type="hat" id="0" value="4" />
+		<input name="hotkey" type="button" id="11" value="1" code="316" />
+		<input name="joystick1left" type="axis" id="0" value="-1" code="0" />
+		<input name="joystick1up" type="axis" id="1" value="-1" code="1" />
+		<input name="joystick2left" type="axis" id="2" value="-1" code="2" />
+		<input name="joystick2up" type="axis" id="3" value="-1" code="5" />
+		<input name="l2" type="axis" id="5" value="1" code="10" />
+		<input name="l3" type="button" id="12" value="1" code="317" />
+		<input name="left" type="hat" id="0" value="8" />
+		<input name="pagedown" type="button" id="8" value="1" code="311" />
+		<input name="pageup" type="button" id="7" value="1" code="310" />
+		<input name="r2" type="button" id="16" value="1" code="706" />
+		<input name="r3" type="button" id="13" value="1" code="318" />
+		<input name="right" type="hat" id="0" value="2" />
+		<input name="select" type="button" id="9" value="1" code="314" />
+		<input name="start" type="button" id="10" value="1" code="315" />
+		<input name="up" type="hat" id="0" value="1" />
+		<input name="x" type="button" id="6" value="1" code="308" />
+		<input name="y" type="button" id="5" value="1" code="307" />
+	</inputConfig>
```

Comment:
- ([rp3][v38] My Stadia controller paired once but didn't work and now isn't repairing without reboot)
- /? stadia
  https://github.com/search?q=repo%3Abatocera-linux%2Fbatocera.linux%20stadia&type=code
- https://github.com/batocera-linux/batocera.linux/pull/7972/files#diff-1cfc7a24cdc48dbc31c3cde5d2d08e973c0191d74a57e9f6ac204066715b3027R3803
  - (Hold asst+capture for 6 seconds to Hard Reset a Stadia Controller)
  - Is it necessary to edit the `deviceName` every time you hard reset and stadia regenerates a new device identifier like StadiaXXXX-YYYY, or is the `deviceGUID` stable across hard resets?

Reply:
(also fwiw, a reddit thread (todo) suggested that stadia controller bt pairing works best when you move joysticks during pairing)